### PR TITLE
script for submodules

### PR DESCRIPTION
**Pull request by TurBoss**

### CODE REVIEW

**Comment by ashdnazg** [comment]
ty!

File: Scripts/update.sh [hunk 1]
```diff
@@ -32,6 +32,7 @@ for REPO in $REPOS; do
 			git checkout master
 			git reset --hard origin/master
 			~/bin/BuildGit "$REPO" "$MODROOT" "$MODINFO" "$PACKAGES/$TAG" "$REMOTE" "$TAG"
+			git submodule update --recursive --remote
```
Comment (abma) [inline Scripts/update.sh]: 1. --init should be added
2. should be run before BuildGit

Comment (ashdnazg) [reply Scripts/update.sh]: oh, oops

Comment (TurBoss) [reply Scripts/update.sh]: thats true ... sorry my fault

**Pull request merged**